File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int arr[] = new int[9];
        int max = 0;
        int index = 0;

        for (int i = 0; i < arr.length; i++) {
            arr[i] = sc.nextInt();
            if (max < arr[i]) {
                max = arr[i];
                index = i + 1;
            }
        }
        System.out.println(max);
        System.out.println(index);
    }
}

File: 백준/Bronze/2562. 최댓값/최댓값.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');
let arr = [];
let max = 0;
let index = 0;

for (let i = 0; i < 9; i++) {
    arr[i] = parseInt(input[i]);
    if (max < arr[i]) {
        max = arr[i];
        index = i + 1;
    }
}

console.log(max);
console.log(index);